tests/src/test/java/org/rhd/katapult/test/services/LoggingConfig.java

```diff
@@ -0,0 +1,44 @@
+package org.rhd.katapult.test.services;
+
+import java.io.InputStream;
+import java.net.URL;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+/**
+ * A logging configuration class that first attempts to load a logging.properties file from the classpath, and then falls
+ * back to a programmatic configuration. To enable, specify -Djava.util.logging.config.class=LoggingConfig in test
+ * app configuration.
+ *
+ */
+public class LoggingConfig {
```

Comment:
Nice; I may eventually move this somewhere it can be used by all the tests.  Configuring the logging in JUL is a PITA. :)